server/app/controllers/dev/DatabaseSeedController.java

```diff
@@ -158,6 +153,44 @@ private QuestionDefinition insertCheckboxQuestionDefinition() {
         .getResult();
   }
 
+  private QuestionDefinition insertCurrencyQuestionDefinition() {
+    return questionService
+        .create(
+            new CurrencyQuestionDefinition(
+                "currency",
+                Optional.empty(),
+                "description",
+                LocalizedStrings.of(Locale.US, "How much should a scoop of ice cream cost?"),
```

Comment:
Not blocking: These references could instead use `LocalizedStrings.withDefaultValue()` to avoid needing to specify `Locale.US`.